packaging/docker/samples/python/fdb.bash

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+#
+# fdb.bash
+#
+# This source file is part of the FoundationDB open source project
+#
+# Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+source /var/fdb/scripts/create_server_environment.bash
+create_server_environment
+source /var/fdb/.fdbenv
+echo "Starting FDB server on $PUBLIC_IP:$FDB_PORT"
+mkdir -p /var/fdb/logs
+fdbserver --listen_address 0.0.0.0:$FDB_PORT --public_address $PUBLIC_IP:$FDB_PORT \
+	--datadir /var/fdb/data --logdir /var/fdb/logs \
+	--locality_zoneid="$(hostname)" --locality_machineid="$(hostname)" --class $FDB_PROCESS_CLASS \
+	--knob_disable_posix_kernel_aio=1
```

Comment:
Forking the start script to pass a single knob seems to suggest a missing feature in the default image. Do you think we should look into supporting knobs at that layer?

This also seems to assume that you're running in an environment that does not support KAIO. I thought that was a specific limitation of Docker for Mac, rather than a more general Docker problem. Is that correct?

Reply:
I do recall looking through the supported environment variables for configuring knobs, but it's unclear if that is supported. That would be ideal over a forked bash script.

AFAICT, there is no reason to include the knob outside of Docker for Mac with M1/arm64, but I stopped short of selectively setting it in this bash script given the observed platform.